src/bsp/src/stm32/src/interloc/src/AngleCalculator.cpp

```diff
@@ -0,0 +1,230 @@
+#include "interloc/AngleCalculator.h"
+#include "interloc/CertitudeCalculator.h"
+#include "interloc/Decawave.h"
+#include <cmath>
+
+constexpr float TWO_PI = 2 * M_PI;
+
+AngleCalculator::AngleCalculator(ILogger& logger) : m_logger(logger) {}
+
+float mod(float val, float div) {
```

Comment:
Why not use the fmod function provided in the math library?

Reply:
Because fmod doesn't really do this. We did try it